src/com/oltpbenchmark/api/BenchmarkModule.java

```diff
@@ -130,6 +130,11 @@ public void createDataSource() throws SQLException {
             props.setProperty("maximumPoolSize", Integer.toString(numConnections));
             props.setProperty("connectionTimeout", Integer.toString(workConf.getHikariConnectionTimeout()));
             props.setProperty("dataSource.reWriteBatchedInserts", "true");
+            if (workConf.getSslCert().length() > 0) {
```

Comment:
Should there be checking that workConf.getSslKey() > 0 at the same time ?

Reply:
Ideally they both should be set..
Will add an assert for the same..